dclab/rtdc_dataset/filter.py

```diff
@@ -114,7 +114,7 @@ def update(self, rtdc_ds, force=[]):
         # Determine which data was updated
         for skey in list(cfg_cur.keys()):
             if skey not in cfg_old:
-                cfg_old[skey] = None
+                cfg_old[skey] = 'none'
```

Comment:
You can drop the `"none"`.
This could be simplified with ``if cfg_cur[skey] != cfg_old.get(skey, None)``.